examples/models/samba_model_example.py

```diff
@@ -83,3 +101,20 @@
 of assistance.
 ===============================================================================
 '''
+
+# Get response information
+response = camel_agent_sambacloud_api.step(user_msg)
```

Comment:
camel_agent_samba_cloud_api

Reply:
Thanks @lightaime , updated in another PR https://github.com/camel-ai/camel/pull/891/commits/50216f276bdfa9ff57c151462c8f5ca5a603f54d